liferay-frontend/liferay-portal
# [7.4-only] LPS-123547 and LPS-102093: Remove components in compatibility layer

Tested and learned from:
https://github.com/liferay-frontend/liferay-portal/pull/505

---

This PR removes all currently deactivated modules in `admin` and `classic` themes compatibility layer.
and removes components_animations

In order to remove all the compatibility layer, we remove all components have been deactivated and tested during last year, from now we only need to remove in `styled` theme.

---

How to test the commit https://github.com/liferay-frontend/liferay-portal/pull/533/commits/3da063296a8b2d77f2f374b0694f136050039dd7:

- Go to the themes directory
- Deploy them
- If it works it's fine, you will not see design changes


How to test the commit https://github.com/liferay-frontend/liferay-portal/pull/533/commits/55cdec3191c18bde013e21eb1368671a27a619eb

1. Open portal an test all collapse and fade animations:
 - Responsive menus
 - Dropdowns
 - SAidebars
